README.md

```diff
@@ -46,6 +46,11 @@ connection is lost and recovered.
 ## Changes from 2.8.0
 * It now uses Guava 25.1-jre version, which might require the client to also upgrade it to match the same version
 
+## Changes from 2.8.10
+* H2 version upgraded to 2.1.210
+* `H2Engine` is now deprecated and uses the H2 legacy mode (for more information see: http://www.h2database.com/html/features.html)
```

Comment:
...the H2 v2 legacy mode if this engine is used with the h2 v2 driver...

Reply:
:+1: